applications/Liberland_judiciary.md

```diff
@@ -0,0 +1,533 @@
+# Liberland Blockchain - the Judiciary
+
+- **Team Name:** Liberland Blockchain Team
+- **Payment Address:** Polkadot 13uikMrP8PvgapAhskyHgM6E8Yw7E6C9LxXMZBpgp2Dm8Nsi
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview :page_facing_up:
+
+Follow-up on Liberland Grant Application [pull request: Create liberland #831; a623bb28f2c9ce9609bf46e0e5801cda6e55aa47] 
+
+### Liberland Blockchain
+
+We are Liberland, the world's freest country. Currently, we are settling the physical land and developing our blockchain. 
+
+Liberland Blockchain is a sovereign blockchain founded and supported by Liberland which provides the first set of use cases. The codebase is a fork of Substrate/Polkadot, with MIT license. Liberland does not claim ownership of the blockchain or related materials.
+
+Functionalities such as trading and DeFi projects are permissionless by default while some critical functionalities such as country politics are permissioned and require KYC.
+
+The next blockchain milestone is translating traditional contract, dispute resolution and judiciary functionalities into the Web3 realm, synergizing the virtues of centralized and decentralized models.
+
+This milestone implements the quintessential element of all governance systems - the judiciary. We seek to enable dispute resolution within this blockchain ecosystem.
+
+### Overview
+
+#### Project Overview
+
+The Liberland Judiciary is developing an on-chain system capable of acting on behalf of a party without their explicit permission in pre-determined ways with mutual consent given in advance.
+
+While cryptocurrencies have successfully managed to build a parallel financial system, other uses of the technology remain rare 
+because its hard to represent the real world or enforce contracts onchain. 
+This anarchic decentralization is at the same time a great benefit but also a big limitation of blockchain technology.
+
+This limitation needs to be solved in order to run a country on the blockchain.
+
+Liberland solves this through courts acting as on-chain oracles providing data and judgements to contracts.
```

Comment:
Can you explain what a court looks like, both on-chain and off-chain? The way I understand it, "acting as on-chain oracle" simply refers to discussing a case IRL and then submitting judgements to the chain, correct? Or is there more involved?